src/status_im/translations/fr.cljs

```diff
@@ -256,59 +236,133 @@
+   :signing-phrase-description            "Signez la transaction en saisissant votre mot de passe. Assurez-vous que les mots ci-dessus correspondent à votre phrase de signature secrète"
+   :wallet-insufficient-funds             "Fonds insuffisants"
+   :request-transaction                   "Effectuer une transaction"
+   :send-request                          "Envoyer la requête"
+   :share                                 "Partager"
+   :eth                                   "ETH"
+   :currency                              "Devise"
+   :usd-currency                          "USD"
+   :transactions                          "Transactions"
+   :transaction-details                   "Détail de la transaction"
+   :transaction-failed                    "transaction échoué"
+   :transactions-sign                     "Signer"
+   :transactions-sign-all                 "Tout signer"
+   :transactions-sign-transaction         "Signer la transaction"
+   :transactions-sign-later               "Signer ultérieurement"
+   :transactions-delete                   "Supprimer la transaction"
+   :transactions-delete-content           "La transaction va être enlevée de la list 'Non-Signé'"
+   :transactions-history                  "Historique"
+   :transactions-unsigned                 "Non-Signé"
+   :transactions-history-empty            "Aucune transaction dans votre historique pour le moment"
+   :transactions-unsigned-empty           "Vous n'avez aucune transaction non signée"
+   :transactions-filter-title             "Filtrer l'historique"
+   :transactions-filter-tokens            "Jetons"
+   :transactions-filter-type              "Type"
+   :transactions-filter-select-all        "Tout sélectionner"
+   :view-transaction-details              "Voir le détail de la transaction"
+   :transaction-description               "Veuillez attendre au moins 12 confirmations pour être sûr que votre transaction a bien été définitivement validée"
+   :transaction-sent                      "Transaction envoyée"
+   :transaction-moved-text                "La transaction va rester dans la liste 'Non-signée' pendant 5 mins"
+   :transaction-moved-title               "Transaction déplacée"
+   :sign-later-title                      "Signer la transaction ultérieurement?"
+   :sign-later-text                       "Vérifier l'historique de la transaction pour la signer"
+   :not-applicable                        "Ne s'applique pas aux transactions non signée"
+
+   ;; Wallet Send
+   :wallet-choose-recipient               "Choisir le destinataire"
+   :wallet-choose-from-contacts           "Choisir à partir des Contacts"
+   :wallet-address-from-clipboard         "Utiliser l'adresse du presse-papiers"
+   :wallet-invalid-address                "Adresse invalide: \n {{data}}"
+   :wallet-browse-photos                  "Parcourir les Photos"
+   :validation-amount-invalid-number      "Le montant n'est pas valide"
```

Comment:
Le montant n'est pas un nombre valide

Reply:
well I thought removing `nombre` makes the phrase lighter without changing the meaning. I'll correct it your way.